src/components/ec-phone-number-input/ec-phone-number-input.vue

```diff
@@ -100,15 +100,15 @@
         class="ec-phone-number-input__number"
         data-test="ec-phone-number-input__number"
         is-in-group="left"
-        type="tel"
+        :type="InputFieldType.TEL"
         :disabled="isDisabled"
         :error-id="errorId"
         :error-message="errorMessage"
         :is-sensitive="isSensitive"
         :placeholder="phoneNumberPlaceholder"
-        :autocomplete="autocomplete"
+        :autocomplete="autocomplete || undefined"
```

Comment:
Because of type of input `autocomplete` attribute. BTW we could get here rid of that `null` value that autocomplete could be. We don't use that value.